auth-api-gms/build.gradle.kts

```diff
@@ -7,7 +7,7 @@ android {
 }
 
 dependencies {
-    api("com.openmobilehub.android:auth-api:1.0.1-beta")
+    api(project(":auth-api"))
```

Comment:
We still want to keep `auth-api` as a separate dependency. Are you having issues? For testing purposes, you can still update gradle dependencies locally.

Reply:
I'm ok with it. But before that I may need to wait for new releases to repos before I don't need to change this line.

And because I moved classes in this PR, inevitably I need to change the dependency here for the code to compile, or else the PR may not be accepted by automated checks🥲

Reply:
Gotcha.. I don't believe there are jobs executing on pull request.

For testing locally, I'd encourage you to use publishToMavenLocal in the meantime we find a longer-term solution.

`./gradlew :auth-api:publishToMavenLocal -x signReleasePublication`

Reply:
Fair enough. Updated PR to use back published artifacts for dependency.

Reply:
Thank you, Raymond for your input on this. I've been looking into it and noticed that when I try to compile, I encounter an Unresolved reference: `nullOrHandled` error. On reviewing the code, it seems to me that the functionality for nullOrHandled might be effectively achieved through a simple function like isHandled instead of an ExperimentalContract. Here's how I think it could be integrated into the EventWrapper class:

```
class EventWrapper<T>(private val content: T) {
    private var hasBeenHandled: Boolean = false

    /**
     * Extracts the content if not handled and marks it as handled.
     *
     * @return null when content has been handled.
     */
    fun getContentIfHandled(): T? {
        if (hasBeenHandled) {
            return null
        }
        hasBeenHandled = true
        return content
    }

    /**
     * Allows you to pick the content without setting it as handled.
     */
    fun peekContent(): T {
        return content
    }

    fun isHandled(): Boolean {
        return hasBeenHandled
    }
}
```
Thoughts?